database/seeds/BodySeeder.php

```diff
@@ -17,26 +17,7 @@ class BodySeeder extends Seeder
      */
     public function run()
     {
-        Address::create([
-            'country_id'    =>  21,
-            'street'        =>  'Notelaarsstraat 55',
-            'zipcode'       =>  '1000',
-            'city'          =>  'Brussels',
-        ]);
-        Address::create([
-            'country_id'    =>  153,
-            'street'        =>  'Oude Markt 24',
-            'zipcode'       =>  '7511 GB',
-            'city'          =>  'Enschede',
-        ]);
-        Address::create([
-            'country_id'    =>  81,
-            'street'        =>  'Bergstraße 66c',
-            'zipcode'       =>  '01069',
-            'city'          =>  'Dresden',
-        ]);
-
-        BodyType::create([
+	BodyType::create([
             'name'          =>  'local',
```

Comment:
Yes, that is actually one of the primary reason we have this, definitely do make the split if you can!